### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by benleelawrence**
Please review the login screen Jason and I created

### CODE REVIEW

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -337,7 +340,7 @@
 				TargetAttributes = {
 					13A342EA1F0367E9002EBC92 = {
 						CreatedOnToolsVersion = 8.3.3;
-						DevelopmentTeam = 5BL5M2XBM2;
+						DevelopmentTeam = LA4QFXA5C5;
```
Comment (dwagg) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj:343]: Did you change the Development team on purpose? Did we need to?

Comment (dwagg) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj:343]: I know this is a generated file but else where it appears the development team was changed

Comment (benleelawrence) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj:343]: I'm not sure if we need this, it wouldn't let me run the program on my iPhone until I changed the Bundle Identifier, I think that's what this is

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
+//
+//  MaxTextLength.swift
+//  Mobile Test Runner for Jama
+//
+//  Created by PSU1 on 7/5/17.
+//  Copyright © 2017 Jaca. All rights reserved.
+//
+
+import Foundation
```
Comment (dwagg) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift:9]: I think this file should be moved into the utils file and out of the viewController file. and we should write a test or two to make sure this is working.

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj (hunk 1 above)
Comment (dwagg) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj:343]: Okay, I'm not sure the best way for all that to happen is, iOS code signing is confusing. But thats fine, it is still working for me, and it runs on your device. I suspect we will end up revisiting this but for now this is good enough for me.

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift (hunk 2 above)
Comment (dwagg) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift:9]: @benleelawrence @JasonRitz I am not sure if you guys saw this comment. What are your thoughts on it. I meant "utils folder" and "ViewControllers" folder in above comment.

Comment (JasonRitz) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift:9]: I moved the file to utils, but don't know what\how to write tests for it. the code adds a max length field in the attributes inspector in Xcode. so it doesn't effect our app but Xcode.

Comment (dwagg) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift:9]: Works for me.

Comment (dwagg) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift:9]: done

**Review by dwagg — CHANGES_REQUESTED** [review]
There is a label on the Login View that says "Log In" but it is not needed as we have to title of the screen in the navigation bar.

**Review by dwagg — APPROVED** [review]

**Comment by mmcbee1** [comment]
Just a suggestion to center the group of input fields: make another view to hold the input fields and login button, then center that vertically/horizontally with the main screen view (centerX and centerY). You would only need to do the leading and trailing margins once for the new view, instead of for each text box, and this would center it for all screen sizes and landscape, too. Then you could pad the space between input fields like we did in the TDD tutorial.

Otherwise great job! Yay we have our first screen!

**Review by mmcbee1 — APPROVED** [review]

**Review by lcc4 — CHANGES_REQUESTED** [review]
This branch is now behind master, so please, in addition to fixing the merge conflict, merge master into this branch and ensure the build still succeeds.  I cannot complete the merge until this happens.

**Review by dwagg — CHANGES_REQUESTED** [review]
Could we move the "instance" text field into the new wrapper view with the other elements?

**Review by dwagg — APPROVED** [review]

**Review by lcc4 — APPROVED** [review]

**Pull request merged**